workspace.spec

```diff
@@ -572,6 +572,378 @@ module Workspace {
+				"CrossRef:IsExpressionOf"
+				"CrossRef:IsFinancedBy"
+				"CrossRef:IsFormatOf"
+				"CrossRef:IsManifestationOf"
+				"CrossRef:IsManuscriptOf"
+				"CrossRef:IsPreprintOf"
+				"CrossRef:IsRelatedMaterial"
+				"CrossRef:IsReplacedBy"
+				"CrossRef:IsReplyTo"
+				"CrossRef:IsReviewOf"
+				"CrossRef:IsSameAs"
+				"CrossRef:IsTranslationOf"
+				"CrossRef:Replaces"
+	*/
+	typedef structure {
+		string id;
+		string description;
+		string relationship_type;
+	} PermanentID;
+
+	/* Title
+		Represents the title or name of a resource.
+
+		The 'title_string' field is required; if no value is supplied for 'title_type', it
+		defaults to 'title'. If the 'title_type' is set to 'translated_title', the appropriate
+		BCP-47 tag must be supplied in the 'title_language' field.
+
+		string title_string - the resource title
+		string title_language (optional) - language that the title is in, as a IETF BCP-47 tag.
+			Examples: 'fr'; 'jp-JP'; 'zh-Hant-CN'; 'en-Latn-GB'; 'mn-Cyrl'.
+		string title_type (optional) - the type of title described; must be one of the following:
+			"title" (default)
+			"subtitle"
+			"alternative_title"
+			"translated_title"
+			"other"
+	*/
+	typedef structure {
+		string title_string;
+		string title_language;
+		string title_type;
+	} Title;
+
+	/* Contributor
+
+		Represents a contributor to the resource.
+
+		Contributors must have a 'contributor_type', either 'person' or 'organization'.
+
+		There must also be an identifier or name of some sort for the contributor.
+
+		If the contributor is an organization, either the 'contributor_id' or
+		the 'name' field must be populated.
+
+		An individual can be represented by either 'given_name' and 'family_name'; a 'name'
+		(if the name is not in the binomial format); or a `contributor_id`, such as an
+		ORCID.
+
+		The 'contributor_role' field takes values from the Datacite and CRediT contributor
+		roles vocabularies. For more information on these resources and choosing the
+		appropriate roles, please see the following links:
+
+		Datacite contributor roles: https://support.datacite.org/docs/datacite-metadata-schema-v44-recommended-and-optional-properties#7a-contributortype
+
+		CRediT contributor role taxonomy: https://credit.niso.org
+
+		string contributor_type - must be either 'person' or 'organization'
+		string contributor_id (optional) - persistent unique identifier for the contributor;
+			e.g. an ORCID for an individual, or a ROR ID for an organization.
+		string given_name (optional) - first name, for individuals whose names can be split
+			into given and surnames
+		string family_name (optional) - family name, for individuals whose names can be
+			split into given and surnames.
+		string name (optional) - contributor name, in cases where the contributor is an
+			organization or a person whose name cannot easily fit into the given/family
+			name structure.
+		list<Organization> affiliations (optional) - list of organizations with which the
```

Comment:
Should this be empty if the contributor is an org?

Reply:
Clarified

Reply:
:+1: